### ISSUE TITLE
Separate variant and size stories in Storybook for Buttons

### ISSUE THREAD

**Issue by kotAPI** [rad-ui/ui#372]
We only have a single storybook entry for size and variants

Where we show sizes and variants all in single story
<img width="1478" alt="image" src="https://github.com/rad-ui/ui/assets/4013556/94222e4e-367d-4f1b-b017-0a4bdff82252">

Let's create a stories for
- All variants of a button in one story
- All sizes for all variants of a button in another story

**Comment by jindaljyoti** [comment]
Yes I will pick that one just let me know what exactly needs to done
